code/modules/projectiles/guns/projectile/pneumatic.dm

```diff
@@ -117,8 +117,12 @@
 
 	if(istype(user, /mob/living))
 		var/mob/living/M = user
-		if ((M_CLUMSY in M.mutations) && prob(50))
-			to_chat(M, "<span class='danger'>[src] blows up in your face.</span>")
+
+		if(M_HULK in M.mutations)
```

Comment:
Could've sworn that was related to hulks in some way, guess not.

Alright then.